web_sanic.py

```diff
@@ -177,6 +177,9 @@ def sighting_to_marker(pokemon, names=POKEMON, moves=MOVES, damage=DAMAGE, trash
         marker['move2'] = moves[move2]
         marker['damage1'] = damage[move1]
         marker['damage2'] = damage[move2]
+    if pokemon.cp:
+        marker['cp'] = pokemon.cp
```

Comment:
Also, this should be "pokemon['cp'] and pokemon['cp_multiplier']
Make sure you are importing your calc_pokemon_level func as well